### ISSUE TITLE
Suggestion: Priority number system

### ISSUE THREAD

**Issue by loneguardian** [hetima/DSP_LSTM#10]
I'd like to suggest a priority number system on top of the cluster system to add more control. It can be incorporated into the name as a second optional parameter (e.g. [C:cluster1:9]). Default priority number is 0 (if nothing is specified) and negative numbers can be used.

Apparently it works differently from the smart system hence the two systems can work in a tier-fashion - e.g. priority number overwrites the distance factor, but when there's a similar priority number situation it will follow the smart system (if it is enabled).

**Comment by hetima** [comment]
I agree that there is potential for expansion in that feature, but it should be carefully considered depending on its practicality and difficulty of implementation.
